src/main/java/org/folio/linked/data/preprocessing/lccn/impl/AbstractSpaceAdder.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.linked.data.preprocessing.lccn.impl;
+
+import static com.github.jknack.handlebars.internal.lang3.StringUtils.SPACE;
+
+import java.util.regex.Pattern;
+import org.folio.linked.data.preprocessing.lccn.LccnNormalizer;
+
+public abstract class AbstractSpaceAdder implements LccnNormalizer<String> {
```

Comment:
minor - `AbstractLccnNormalizer` is a better name I think.
Similarly, `LccnNormalizerStructrueA` and `LccnNormalizerStructrueB`

Reply:
done